scripts/mirror.js

```diff
@@ -51,6 +51,13 @@ const { argv } = require('yargs').command(
           'Specify when to perform mirroring, whether on true/null ("nonreal", default), true/null/false ("bool"), or always ("always")',
         type: 'string',
         default: 'nonreal',
+      })
+      .option('target_version', {
```

Comment:
Looking at `git`s and `yarn`s command line options, a dash seems to be common to separate words in options rather than underscores:

```suggestion
      .option('target-version', {
```